apps/openmw/mwworld/projectilemanager.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 
+#include <osg/observer_ptr>
```

Comment:
I'm using clang version 3.6.2-1, but I don't know if that will make a difference.